packages/embeds/embed-core/src/embed.ts

```diff
@@ -240,6 +240,7 @@ export class Cal {
     const iframe = (this.iframe = document.createElement("iframe"));
     iframe.className = "cal-embed";
     iframe.name = `cal-embed=${this.namespace}`;
+    iframe.title = `cal-embed`;
```

Comment:
Let's go with this
```suggestion
    iframe.title = `Book a call`;
```

Reason being embed is self-hosted as well and we might need to use the self-hosted name of the application, so I want to avoid Cal.com for now.

Accessiblity wise I think **Book a Call** serves the purpose.